src/GovernorCountingFractional.sol

```diff
@@ -3,6 +3,9 @@
 
 pragma solidity ^0.8.0;
 
+// Disabling forgefmt to stay consistent with OZ's style.
```

Comment:
Is this because we plan to get this upstreamed into OZ?

Reply:
Exactly